cdmi-stub/mediakeysession.cpp

```diff
@@ -154,9 +158,15 @@ void CMediaKeySession::Update(
   CDMI_DLOG() << "#mediakeysession.Run" << endl;
   std::string key_string(reinterpret_cast<const char*>(f_pbKeyMessageResponse),
                                    f_cbKeyMessageResponse);
+#if WPE
+  std::string key_id("1", 1);
+  media::KeyIdAndKeyPair key(key_id, key_string);
+  g_keys.assign(1, key);
+#else  //Chrome      
```

Comment:
Can you explain why these changes are required?

If we really do have to have browser specific parts (which I'm not convinced we do) in the CDMi layer it needs to be decided on at run time (not via #ifdef). Maybe the browser agent string needs to be sent or similar.

Reply:
So far the URL we tested, does not sent the key in JSON format. That why there is no need to extract key from json.